man/fi_verbs.7.md

```diff
@@ -170,6 +170,14 @@ The verbs provider checks for the following environment variables.
 : The prefix or the full name of the network interface associated with the verbs
   device (default: ib)
 
+*FI_VERBS_MR_CACHE_ENABLE*
+: Enable Memory Region caching (default: 0)
+
+*FI_VERBS_MR_CACHE_SIZE*
+: Maximum number of cache entries (default: 4096)
+
+*FI_VERBS_MR_CACHE_LAZY_SIZE*
+: Minimum size of lazy deregistration (default: 0)
```

Comment:
Please expand on the documentation.  Unless someone is familiar with verbs and what a MR cache is, they won't really know what these fields do or the affect that they have.  I would describe what a MR cache actually is, and provide more details on what a 'lazy size' is, in particular.